scripts/ci.sh

```diff
@@ -70,17 +70,20 @@ install_qemu() {
 	MF_PATH=$GOPATH/src/github.com/mainflux/mainflux
 	cd $MF_PATH
 	sudo apt-get update && sudo apt-get -y install qemu-user-static
-	wget https://github.com/multiarch/qemu-user-static/releases/download/v2.11.1/qemu-arm-static.tar.gz  && tar -xzf qemu-arm-static.tar.gz && rm qemu-arm-static.tar.gz
-	sudo cp qemu-arm-static /usr/bin/
+	wget https://github.com/multiarch/qemu-user-static/releases/download/v2.11.1/qemu-arm-static.tar.gz  \
+		&& tar -xzf qemu-arm-static.tar.gz \
+		&& rm qemu-arm-static.tar.gz
+	sudo mv qemu-arm-static /usr/bin/
```

Comment:
`mqtt/Dockerfile.arm` demands this file in mainflux root, so you can't move it from here, at least without other modifications